journal.py

```diff
@@ -17,10 +17,8 @@
 
 from peewee import *
 
-
-# TODO: make the database point to a seperate home directory for the specific OS
-db = SqliteDatabase('diary.db')
-
+path = os.environ['HOME'] + '/.tnote'
```

Comment:
The code works fine on both my Windows 10 machine and my Ubuntu Precise machine.